include/wx/valtext.h

```diff
@@ -19,6 +19,9 @@ class WXDLLIMPEXP_FWD_CORE wxTextEntry;
 
 #include "wx/validate.h"
 
+#define wxEVT_TXT_VALIDATE wxEVT_VALIDATE
```

Comment:
We use `TEXT` rather than `TXT` everywhere else, so we should do it here.

Provided that we actually need these constants at all -- what's wrong with just using `wxEVT_VALIDATE` directly?